lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -1566,9 +1568,13 @@ LogicalResult FIRRTLLowering::visitDecl(WireOp op) {
   if (resultType.isInteger(0))
     return setLowering(op, Value());
 
-  // Name attr is requires on sv.wire but optional on firrtl.wire.
+  // Name attr is required on sv.wire but optional on firrtl.wire.
   auto nameAttr = op.nameAttr() ? op.nameAttr() : builder.getStringAttr("");
-  return setLoweringTo<sv::WireOp>(op, resultType, nameAttr);
+  auto symName = op.nameAttr() ? op.nameAttr() : StringAttr();
```

Comment:
I think you just want `auto symName = op.nameAttr();` instead of the ternary.

Reply:
Eventually we should only add this symbol when there is a "dont touch" attribute present at the FIRRTL level.

Reply:
> 
> 
> I think you just want `auto symName = op.nameAttr();` instead of the ternary.

Updated to prepend the module name to the symbol name, we need it to try and make the symbol name unique outside the module in the symbol table.

Reply:
> 
> 
> Eventually we should only add this symbol when there is a "dont touch" attribute present at the FIRRTL level.

Sure, was planning create a separate PR to  handle the "dont touch" annotation